cdap-app-fabric/src/main/java/co/cask/cdap/metadata/serialize/RelationProto.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.metadata.serialize;
+
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * Class to serialize {@link co.cask.cdap.data2.metadata.lineage.Relation}.
+ */
+public class RelationProto {
```

Comment:
``LineageProto`` is the output of the lineage REST API, though, right? Its ok to do it in another PR, but we should do it in 3.2?